### DISCUSSION TITLE
Twitch.Tv videos are no longer detected.

### DISCUSSION THREAD

**Discussion by TecAgnes** [aclap-dev/video-downloadhelper#850] [Help and Q&A]
### Your question

Hello all,

I used to download many videos on Twitch.Tv for months, I have even paid for the license especially for it at first, but lately, it does not work anymore, videos are not detected. I've tried the extension on Youtube and it seems to work fine there. It is like Twitch.TV is no longer supported by Video DownloadHelper.

Do you have any idea if this is a general issue and if it can be solved is some ways?

Thanks a lot for your help,

Regards,

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: fr
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"3cf0ec8457694878"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "V:\\",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_legacy_ui": false,
    "use_sidebar": true,
    "open_count_store": 4
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3672, Firefox 126.0.1 64-bit, licensed VDH 9.0.2.3a1 beta, CoApp 2.0.19.

I went to the Twitch home page & VDH offered me a 28 second item to download.  I thought that sounded pointless but I decided to see what I would get.  After downloading the 28 seconds & showing 100% complete, the VDH status display changed & no longer showed a duration nor a percentage complete.  It just continued to download.  After about a minute, I stopped it.  I had an MP4 that VLC was perfectly happy to play.  So I can't reproduce your results.  It works fine for me.

Maybe this is yet another example of how Firefox works better than Chrome.  Maybe.  I'm just guessing.  Like I said, I'm not a developer.  You might also try the beta to see if that improves things.  I've never used Chrome so I can't say how likely it is that the VDH beta for Chrome will make any difference.  But you could try it & see.  I would never suggest you dump Chrome & start using Firefox instead.  That would be rude.  I'm a polite guy.

**Reply by TecAgnes** [reply]
Thank you very much for your time and details!

I will try the beta version indeed, this is a good idea! And I will even try the Firefox version, I agree VDH could work much better with it. And this is my own decision of course, not based on any suggestion, thanks for being so polite 😁  

I will post another comment later after my test.

Thanks again!

**Comment by TecAgnes** [comment]
Hello again,

Beta version is not working better on Chrome, but using Firefox has definitely solved my problem. Thanks again for your help!

The only thing is my VDH license is not accepted on Firefox as it is a Chrome one, so I might have to buy another license for it. I will see if I can convert the one I own instead.

Thanks !

**Reply by GrampaWildWilly** [reply]
Converting isn't possible.  However, if you bought your Chrome license within the past month, @jcfrog Jérôme would be willing to refund it.  You just have to follow the instructions in the Licenses section of the Wiki.  But then you would be without a Chrome VDH license, even if you did get a Firefox VDH license.  Are you OK with that?  Are you OK with switching back & forth between the 2 browsers?  Because that's what you would have to do.  You could still download using VDH for Chrome but some unknown number of those downloads would have the watermark, which can take an unreasonable time to write.  Plus on Chrome, there's a restriction of only 1 download every 2 hours.  It depends on whether you think you can afford both licenses.  Remember, the license is for life, on all the computers you have now & all the computers you will ever have.  So even with 2 licenses, they get cheaper as each day passes.  Things to think about.

**Reply by TecAgnes** [reply]
Thanks again for your help and sorry for the late reply!

I think I will buy a new license indeed, for all good reasons you have mentioned, plus it is always a pleasure to support such good creations when we can. This is not expensive at all and provide a great help for our needs.

Have fun !
